.github/workflows/cmake.yml

```
@@ -0,0 +1,35 @@
+name: CMake build and test
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '*.md'
+      - 'docs/**'
+  pull_request:
+    types: [ opened, synchronize ]
+    paths-ignore:
+      - '*.md'
+      - 'docs/**'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

I noticed that there is a now inactive Travis setup where macOS builds used to be enabled https://github.com/Tehreer/SheenBidi/blob/master/.travis.yml#L4 Maybe it would be good to added macOS here so the project can be tested on both Linux and macOS again.

```suggestion
    strategy:
      matrix:
        os: [ubuntu-latest, macos-latest]
    runs-on: ${{ matrix.os }}
```

---

Done